lib/hanami/generators/app/application.rb.tt

```diff
@@ -85,20 +94,24 @@ module <%= config[:classified_app_name] %>
       #
       # default_request_format :html
 
-      # Default format for responses that doesn't take into account the request format
+      # Default format for responses that doesn't take into account the request
+      # format
```

Comment:
97 and 98 can be combined into one line by rewording to:  
`    # Default format for responses that don't consider the request format`